PyRDF/__init__.py

```diff
@@ -19,4 +20,25 @@ def use(backend_name, conf = {}):
     elif backend_name == "spark":
         current_backend = Spark(conf)
     else:
-        raise Exception(" Incorrect backend environment \"{}\"".format(backend))
\ No newline at end of file
+        raise Exception(" Incorrect backend environment \"{}\"".format(backend))
+
+def include(includes_list):
+    """
+    Includes a list of C++ headers to be
+    declared before execution.
+
+    parameters
+    ----------
+    includes_list : list or str
+        This list should consist of all necessary C++
+        headers as a string. This parameter defaults
```

Comment:
The parameter no longer defaults as an empty list

Reply:
Will fix this @etejedor 👍